gatsby/src/pages/drugs/{BnfDrug.slug}/medicinal-forms.tsx

```diff
@@ -54,27 +88,61 @@ const MedicinalFormsPage: FC<MedicinalFormsPageProps> = ({
 			id: slug,
 			title: form,
 		}))}
+		asideContent={asideInfo}
+		headerCta={
+			<Link to={`/drugs/${slug}/`}>
+				View monograph<span className="visually-hidden"> for {title}</span>
+			</Link>
+		}
 	>
 		<p dangerouslySetInnerHTML={{ __html: initialStatement }} />
 		{specialOrderManufacturersStatement && (
 			<p
 				dangerouslySetInnerHTML={{ __html: specialOrderManufacturersStatement }}
 			/>
 		)}
-		{medicinalForms.map(({ form, slug, preps }) => (
-			<section key={form} aria-labelledby={slug}>
-				<h2 id={slug}>{form}</h2>
-				{preps.length ? (
-					<ol>
-						{preps.map((prep) => (
-							<li key={prep.ampId}>
-								<Prep prep={prep} />
-							</li>
-						))}
-					</ol>
-				) : null}
-			</section>
-		))}
+		{medicinalForms.map(
+			({ form, slug, preps, cautionaryAndAdvisoryLabels }) => (
+				<section className={styles.form} key={form} aria-labelledby={slug}>
+					<h2 id={slug}>{form}</h2>
+					{cautionaryAndAdvisoryLabels?.length ? (
+						<Accordion
+							className={styles.labelAccordion}
+							theme={AccordionTheme.Warning}
+							title={
+								<span className={styles.labelAccordionHeading}>
```

Comment:
This feels like _maybe_ it should be a `h3` as it's the heading for the list below? Possibly even could have an id to use as the labelled-by for the list?